apps/src/code-studio/components/progress/UnitOverviewHeader.jsx

```diff
@@ -114,6 +137,11 @@ class UnitOverviewHeader extends Component {
       showRedirectWarning &&
       !dismissedRedirectWarning(courseName || scriptName);
 
+    // Get if the user is using an outdated script
+    const displayOutdatedCourseWarning = Object.keys(
+      OUTDATED_SCRIPT_NAMES_AND_DESC
+    ).includes(scriptName);
```

Comment:
Another nit that is totally up to you. Simply looking up the key in the `OUTDATED_SCRIPT_NAMES_AND_DESC` object should get the job done being that `undefined` will be returned.

Something like this would work (Also optional `!!` to maintain a `boolean` result):

`!!OUTDATED_SCRIPT_NAMES_AND_DESC[scriptName]`